app/learning/index.md

```diff
@@ -5,204 +5,98 @@ title: Getting started with Yeoman
 sidebar: sidebars/learning.md
 ---
 
-The Yeoman workflow is comprised of three core tools for improving your productivity and satisfaction when building a web app. These tools are:
+Yeoman is a generic scaffolding system allowing you to create any kind of app. It'll allow you to rapidly get started on new projects and streamline the maintenance of existing projects.
```

Comment:
`Yeoman is a generic scaffolding system allowing the creation any kind of app. It allows for rapidly getting started on new projects and streamlines the maintenance of existing projects.`